app/controllers/concerns/rate_limit_concern.rb

```diff
@@ -1,9 +1,12 @@
 module RateLimitConcern
   extend ActiveSupport::Concern
 
-  def confirm_not_rate_limited
+  # idv_resolution idv_doc_auth proof_address proof_ssn
+  ALL_IDV_RATE_LIMITTERS = [:idv_resolution, :idv_doc_auth, :proof_address, :proof_ssn].freeze
```

Comment:
Probably don't need the comment with the list of rate limiters any more.

Since a user can complete IdV via verify by mail, maybe we shouldn't exclude them from the rest of the flow if they're rate limited at the phone step (:proof_address). Currently they can't cancel their letter and start over if they're rate limited at the phone step.

Reply:
>  maybe we shouldn't exclude them from the rest of the flow if they're rate limited at the phone step (:proof_address)

What does this look like? Is it an option to verify by mail on the error message the user sees or getting rid of `confirm_not_rate_limited_after_idv_resolution` all together?

Reply:
I was thinking that :proof_address wouldn't be included in these checks until you get to phone step. There was a similar issue with the review step that Amir fixed.

Reply:
I see. Let me see what I can do